packages/react-native/ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedModule.java

```diff
@@ -149,7 +148,7 @@ public long getBatchNumber() {
   }
 
   private class ConcurrentOperationQueue {
-    private final Queue<UIThreadOperation> mQueue = new ConcurrentLinkedQueue<>();
+    private final Queue<UIThreadOperation> mQueue = new LinkedBlockingQueue<>();
```

Comment:
Why exactly does it solve the crash?

Also, what kind of crash? Is there an issue reported?

Reply:
Actually I don`t know why this change can fix this issue. Maybe I just proposed this issue and need to further  check